content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,170 @@
+3. Define the rule scope. The rule scope defines when the rule should be evaluated. For example, you can create a rule that it is evaluated only on specific repositories and branches. To define the scope for a rule, switch to `select when to evaluate` and add included or excluded values for the scope. More information on [rule scopes](#rule-scope).
+4. Define the rule condition. The rule condition states in which scenario the rule fails, failing the related pipeline (if the rule is blocking). You can select one of the existing rule conditions for the rule type you have selected.
+
+   The following example shows how to create a static analysis rule that will fail when there are one or more static analysis violations with "error" status and "security" category being introduced in a specific commit:
+
+   {{< img src="ci/qg_rule_condition_sa_errors_security.png" alt="Rule for static analysis security errors" style="width:80%;">}}
+
+5. Select whether the rule should block the pipeline or not when it fails. Non-blocking rules are still evaluated, but
+they do not block the pipeline when they fail.
+6. Select the time window over which the query will be evaluated.
+7. Select a rule name that describes the rule that you are creating.
+8. Click **Save Rule**.
+
+
+## Datadog-ci gate evaluate
+
+To use quality gates, [`datadog-ci`][7] version should be higher or equal than `2.16.0`.
+
+You can invoke the Quality Gates evaluation by calling the [`datadog-ci gate evaluate`][4] command.
+
+This command:
+
+1. Retrieves all the rules that have [rules scopes](#rule-scope) and [custom scopes](#custom-scope) matching the current pipeline context (repository, branch, or custom scope passed in the command).
+2. Evaluates all the matching rules.
+3. If one or more blocking rules fail, the command fails as well, blocking the pipeline.
+
+<div class="alert alert-danger"><strong>Note:</strong> for the command to work properly, it's important that the events (tests, static analysis violations)
+are sent to Datadog <strong>before</strong> the <code>datadog-ci gate evaluate</code> command is executed.
+Otherwise, the rules might have an incorrect behavior due to the absence of the events.
+</div>
+
+The command requires the `DATADOG_API_KEY` and `DATADOG_APP_KEY` environment variables to point to your [Datadog API Key][5]
+and [Datadog Application Key][6]. Also, you need to set the `DD_BETA_COMMANDS_ENABLED` environment
+variable as `true`. Optionally, you can specify the `DATADOG_SITE` environment variable to point to a specific datadog site (default value is `datadoghq.com`).
+
+{{< code-block lang="shell" >}}
+DD_BETA_COMMANDS_ENABLED=true DATADOG_SITE={{< region-param key="dd_site" >}} DATADOG_API_KEY=<API_KEY> DATADOG_APP_KEY=<APP_KEY> datadog-ci gate evaluate
+{{< /code-block >}}
+
+The behavior of the command can be modified using the following flags:
+- --fail-on-empty: when this flag is specified, the command fails if no matching rules were found in Datadog
```

Comment:
You only specify the flag? Or do you need to set it as true?
Another thing, be aware that the double hyphen is being replaced by a dash.

Reply:
You only specify the flag. Yes, I have [asked](https://github.com/DataDog/documentation/pull/18954#discussion_r1266978293) to @alai97 about this, we are trying to remove the bolding to see if that changes something

Reply:
if we use the code markdown, will the double hyphen not be replaced?

Reply:
It's working now